native/core/src/parquet/objectstore/s3.rs

```diff
@@ -169,19 +169,19 @@ fn extract_s3_config_options(
     }
 
     // Extract and handle path style access (virtual hosted style)
```

Comment:
```suggestion
    // Set 'virtual hosted style' config depending on the value of 'path style access'
```
I am not sure that it is more clear